charts/memgraph/templates/statefulset.yaml

```diff
@@ -4,53 +4,60 @@ kind: StatefulSet
 metadata:
   name: {{ include "memgraph.fullname" . }}
   labels:
-    app.kubernetes.io/name: memgraph
-    app.kubernetes.io/managed-by: Helm 
+    {{- include "memgraph.labels" . | nindent 4 }}
+  {{- with .Values.podAnnotations }}
```

Comment:
This should me moved under spec.template.metadata.annotations,
and then have a separate statefulSet.annotations values